src/main/solr/dssolr/conf/schema.xml

```diff
@@ -661,10 +661,33 @@ In solrconfig.xml the classic schema is defined with this:  <schemaFactory class
     <?description ?>
     <?example     ?>
   </field>
-  <!--TODO: Figure out what this field can contain-->
-  <field name="internal_program_structure_overlaps" type="string">
-    <?description ?>
-    <?example     ?>
+  <field name="internal_program_structure_overlaps" type="boolean">
+    <?description A boolean value that indicates if there are overlaps between this resource and other resources.?>
```

Comment:
What is an overlap? I'm guessing it is time based and probably coupled to wall time? Don't be afraid to use more words!